packages/apputils/src/mainareawidget.ts

```diff
@@ -45,12 +45,20 @@ export class MainAreaWidget<T extends Widget = Widget>
     toolbar.node.setAttribute('role', 'navigation');
     toolbar.node.setAttribute('aria-label', trans.__('notebook actions'));
     const spinner = this._spinner;
+    const customHeader = (this._customHeader =
+      options.customHeader ||
```

Comment:
Maybe the `??` operator would also be appropriate here?

```suggestion
      options.customHeader ??
```

Reply:
I think we wouldn't be opposed to you changing the other `||` in the several places above to `??` either.

Reply:
I had thought that the reasoning behind using `||` was that if the caller passed in falsey `toolbar`, for example, the function would construct a valid one, which seemed like a good idea. If we used `??`, a caller can provide `{toolbar: false}` or `{customHeader: false}`, which would then be stored as-is in the object and cause lots of problems later?

Reply:
1. Good point that switching || to ?? is technically a backwards-incompatible change
2. Typically what we are going for here are optional values that are not passed in (so undefined, not false). However, if you want to keep things consistent until we can change things in a way, I think that's fine too.

Reply:
Right, because I can't be sure who is passing in `false` here for `toolbar`, I hesitate to make this change, especially since it's not naturally related to the broader PR?